docs/secure-connections/sauce-connect/advanced/kgp.md

```diff
@@ -1,50 +1,47 @@
 
-## KGP - Sauce Connect Proxy&trade; Tunneling Protocol
 
-Sauce Connect Proxy&trade; establishes an encrypted connection between
-[Sauce Connect Proxy&trade; client](/secure-connections/sauce-connect/advanced/architecture/#sauce-connect-proxy-client) and
-[Sauce Connect Proxy&trade; server](/secure-connections/sauce-connect/advanced/architecture/#sauce-connect-proxy-server).
-This TCP connection is used a reverse tunnel between a customer environment and Sauce Labs data center.
-The protocol that is used to achieve the reverse tunneling is called KGP.
+## About KGP Tunneling Protocol
 
-KGP is an application layer protocol that carries all HTTP(s) traffic as its payload.
-It is developed and maintained by Sauce Labs, and is used to multiplex established connections for multiple HTTP requests/responses.
+Sauce Connect Proxy establishes an encrypted TCP connection between the [Sauce Connect Proxy client](/secure-connections/sauce-connect/advanced/architecture/#sauce-connect-proxy-client) and [Sauce Connect Proxy server](/secure-connections/sauce-connect/advanced/architecture/#sauce-connect-proxy-server). This connection is used as a reverse tunnel between the user environment and Sauce Labs data center.
+
+The protocol used to achieve the reverse tunneling is called _KGP_. It's an application layer protocol that carries all HTTP(s) traffic as its payload.
+Developed and maintained by Sauce Labs, KGP is used to multiplex established connections for multiple HTTP requests/responses.
 
 KGP is preferred over conventional protocols (such as reverse SSH tunnel) for a number of reasons:
+- It’s lightweight
+- It reconnects when a connection accidentally disconnects
+- It ensures that all the data is sent and received, even over an unstable or intermittent connection
+- It provides information about the connection state
+
 
-    - It’s lightweight
-    - It reconnects when a connection accidentally disconnects
-    - It ensures that all the data is sent and received, even over an unstable or intermittent connection
-    - It provides information about the connection state
+## KGP Packet Types
 
-## KGP Packet types
+KGP defines the following packet types:
 
-KGP defines the following packet types
+- Data packets carrying HTTP(s) traffic
+- Control packets, which include:
+  - Connection requests
+  - Keepalive timers
 
-    - Data packets carrying HTTP(s) traffic
-    - Control packets, which include:
-    - Connection requests
-    - Keepalive timers
 
 ## KGP Security
 
-KGP, by itself, is not responsible for the connection security, it just allows to multiplex an existing TCP connection.
-However, all KGP packets are encrypted with the industry standard TLS 1.2 protocol using OpenSSL library.
+KGP, by itself, is not responsible for the connection security. It just allows to multiplex an existing TCP connection. However, all KGP packets are encrypted with the industry standard TLS 1.2 protocol using OpenSSL library.
```

Comment:
@waggledans should this be "It just allows _you_ to multiplex an existing TCP connection."